re/service.go

```diff
@@ -395,8 +395,10 @@ func (re *re) generateReport(ctx context.Context, cfg ReportConfig, action Repor
 		sMsgs := []senml.Message{}
 
 		pm.Offset = uint64(0)
-		pm.Publisher = metric.ClientID
 		pm.Name = metric.Name
+		if metric.ClientID != "" {
```

Comment:
@nyagamunene  Could please test with Report for "channel 1" for "temp"   
where  "Client 1" and "Client 2" both publishes "temp" to "Channel 1"